app/utility/base_knowledge_svc.py

```diff
@@ -107,8 +109,13 @@ async def _update_relationship(self, criteria, updates):
         matches = await self._get_relationships(criteria)
         for match in matches:
             for k, v in updates.items():
-                if getattr(match, k, False):
-                    setattr(match, k, v)
+                if getattr(match, k, "eMpTy") is not "eMpTy":
```

Comment:
it seems the failing flake8 is here for `is not` and it should be `!=`

Reply:
Good catch, thanks!